### PR TITLE
Added rules to the website so we can get rid of the forum post

### PR DESCRIPTION
**Pull request by louvenarde**
Added rules to the website so we can get rid of the forum post
Spoilers are gone :frowning_face: But it's okay. Overall I think it's better, cause we can have everything in the one central place, and also since the rules are on git we'll get a "rule changelog" with the commit history :slightly_smiling_face: 

image

image

### CODE REVIEW

File: templates/views/rules.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+Avoiding penalties applied to you by the moderation team via any means is forbidden, and will result in an extension of the penalties.
+Helping others with breaking FAF rules or avoiding penalties is also forbidden, and carries a similar punishment.
+For example, sharing an account or playing with a banned user who is avoiding their penalty will result in a ban of both users.
+
+## In-game Rules:
+**All of the rules for FAF apply ingame**, excluding the english language requirement.
+
+- You will **not exploit any bugs or loopholes** in any of the services. You can find a list of exploits here: [exploits](http://forums.faforever.com/viewtopic.php?f=2&t=11322)
+
+- If you discover any exploit report it to tech support: viewtopic.php?f=3&t=12531
```
Comment (norraxx) [inline templates/views/rules.md]: Will this link work?

File: templates/views/rules.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+- Discussing or offering pirated material, or links to such.
+- Messages containing copyrighted material unless that material is owned by the sender or by FAF.
+- Discussing acts of illegal nature.
+- Discussing moderation decisions in public is discouraged. Channels exist for appealing processes and raising concerns about moderation.
+- Using usernames that are offensive or mimicking other FAF users.
+- Impersonating a member of the community staff or any FAF user, or breaking bots by taking their names.
+- Spam or content of a spammy nature, and material that degrades the quality of the communication medium.
+
+The userbase for this lobby is international in nature, as such #aeolus is generally reserved for English chat. Quick questions or brief conversations in other languages are allowed, but if asked to stop, they should quickly.
+There are language channels for **#russian #german** and **#french**.
```
Comment (norraxx) [inline templates/views/rules.md]: Comma?

File: templates/views/rules.md (hunk 1 above)
Comment (louvenarde) [reply templates/views/rules.md]: oops, no it wont

File: templates/views/rules.md (hunk 2 above)
Comment (louvenarde) [reply templates/views/rules.md]: yup

**Pull request merged**